dev/devicelab/lib/tasks/web_benchmarks.dart

```diff
@@ -0,0 +1,166 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
question: are we planning to run the benchmarks in any other browser soon (meaning in a few quarters) if so I will strongly argue that we should abstract the browser related dependencies to another class. It took a while to make felt modular for example. This time we should start with a flexible code from the start :)

Reply:
In the short-term - Q1-Q2 - the plan is to only run on Chrome. However, when we are ready to add more browsers, I think we should find a way to share code with `felt`.